src/parser/jobs/drg/modules/BloodOfTheDragon.js

```diff
@@ -274,40 +410,65 @@ export default class BloodOfTheDragon extends Module {
 				return <StatusLink key={id} showName={false} iconSize="35px" {...getDataBy(STATUSES, 'id', id)} />
 			})
 
-			return <Table.Row key={cast.timestamp}>
+			return <Table.Row key={cast.timestamp} warning={cast.action.id === ACTIONS.STARDIVER.id && window.missedSdBuff}>
 				<Table.Cell>{this.createTimelineButton(cast.timestamp)}</Table.Cell>
 				<Table.Cell><ActionLink {...cast.action} /></Table.Cell>
 				<Table.Cell>{buffs}</Table.Cell>
 			</Table.Row>
 		})
 
-		const buffsInDelayWindow = {}
-		let canBeDelayed = window.activeBuffs.length === 0
-		let couldBeDelayed = false
-
-		for (const id in window.timeToNextBuff) {
-			buffsInDelayWindow[id] = window.timeToNextBuff[id] >= LOTD_BUFF_DELAY_MIN && window.timeToNextBuff[id] <= LOTD_BUFF_DELAY_MAX
-			couldBeDelayed = buffsInDelayWindow[id] || couldBeDelayed
-			canBeDelayed = window.timeToNextBuff[id] >= LOTD_BUFF_DELAY_MIN && canBeDelayed
-		}
-
-		const delayBuffs = Object.keys(buffsInDelayWindow).filter(id => buffsInDelayWindow[id]).map((id, idx) => {
+		const delayBuffs = Object.keys(window.buffsInDelayWindow).filter(id => window.buffsInDelayWindow[id]).map((id, idx) => {
 			const action = getDataBy(ACTIONS, 'id', parseInt(id))
 			return <Message.Item key={idx}><Trans id="drg.blood.delay-buff"><ActionLink {...action} /> in {this.parser.formatDuration(window.timeToNextBuff[id])}</Trans></Message.Item>
 		})
 
 		return <Fragment>
-			{canBeDelayed && couldBeDelayed && (
+			{window.isLast && (
+				<>
```

Comment:
nit; only need fragments (`<>...</>`) if you're wrapping multiple top-level JSX elements (the act as an invisible "root" node, as you always need one). when you do need them, they have the same structural effect as parens.

should only ever need parenthesis _or_ fragments, not both - in this case, parens should be fine